introduction.md

```diff
@@ -1,3 +1,9 @@
 Hello, Shama here.
 
-Favorite Code School path: JavaScript
\ No newline at end of file
+Favorite Code School path: JavaScript
+
+Deadly Skills
+
+* Git 
+* JavaScript 
+*Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?